### ISSUE TITLE
Automated test suite

### ISSUE THREAD

**Issue by jayvdb** [tinloaf/tableprinter#1]
It would be helpful to have an automated test suite, to check this works correctly on new versions of Python, etc.

**Comment by tinloaf** [comment]
I agree; I've written it on my (long…) To-Do-list. Thanks for the suggestion!
